open-alexander/src/main/java/us/ihmc/alexander/parameters/controller/AlexanderHighLevelControllerParameters.java

```diff
@@ -7,7 +7,6 @@
 
 import us.ihmc.alexander.AlexanderJointMap;
```

Comment:
I would like to see this class get copied into the AlexanderSoftware repo, and this class be renamed to OpenAlexander, so that we're not tuning across repos. Same thing for basically everything in the controller project here.

Reply:
this is the conversation we had yesterday in the meeting, and why I'm half thinking of renaming the robot that's contained in open robotics to just avoid the confusion. 

We don't care about updating the stuff contained in open-robotics, because that's not what the robot runs from. it is meant to have the tests for the controller be contained next to the controller, as a representative system, rather than the true thing.

Reply:
and remove open alexander